lib/commons/aria/get-accessible-refs.js

```diff
@@ -0,0 +1,25 @@
+import getRootNode from '../dom/get-root-node';
+import cache from '../../core/base/cache';
+import cacheIdRefs from './cache-idrefs';
```

Comment:
Suttest putting that cachedIdRefs into this file. I don't think that's a function that should ever be available anywhere else.